File: jscomp/test/key_word_property2.js
'use strict';

Export_keyword = require("./export_keyword.js");

function test2(v) do
  return do
          open: v.open,
          window: v.window
        end;
end

function test(p) do
  return --[ tuple ]--[
          p.catch,
          p.then
        ];
end

$$case = Export_keyword.$$case;

$$window = Export_keyword.$$window;

$$switch = Export_keyword.$$switch;

exports.test2 = test2;
exports.test = test;
exports.$$case = $$case;
exports.$$window = $$window;
exports.$$switch = $$switch;
--[ No side effect ]--

File: jscomp/test/key_word_property2.lua
__console = {log = print};

Export_keyword = require "..export_keyword";

function test2(v) do
  return {
          open = v.open,
          window = v.__window
        };
end end

function test(p) do
  return --[[ tuple ]]{
          p.__catch,
          p.__then
        };
end end

__case = Export_keyword.__case;

__window = Export_keyword.__window;

__switch = Export_keyword.__switch;

exports = {};
exports.test2 = test2;
exports.test = test;
exports.__case = __case;
exports.__window = __window;
exports.__switch = __switch;
return exports;
--[[ No side effect ]]
